docs/content/doc/actions/act-runner.en-us.md

```diff
@@ -0,0 +1,211 @@
+```
+
+And you will be asked to input the registration information step by step. Includes:
+
+- The Gitea instance URL, like `https://gitea.com/` or `http://192.168.8.8:3000/`.
+- The registration token.
+- The runner name, which is optional. If you leave it blank, the hostname will be used.
+- The runner labels, which is optional. If you leave it blank, the default labels will be used.
+
+You may confuse about the runner labels, we will explain it later.
+
+If you want to register the runner in a non-interactive way, you can use arguments to do it.
+
+```bash
+./act_runner register --no-interactive --instance <intance_url> --token <registration_token> --name <runner_name> --labels <runner_labels>
+```
+
+When you have registered the runner, you can find a new file named `.runner` in the current directory.
+This file stores the registration information.
+Please do not edit it manually.
+If this file is missing or corrupted, you can simply remove it and register again.
+
+If you want to store the registration information in another place, you can specify it in the configuration file,
+and don't forget to specify the `--config` option.
+
+### Register the runner with docker
+
+If you are using the docker image, the things are a little different.
+Registration and running are combined into one step, so you need to specify the registration information when running the act runner.
+
+```bash
+docker run \
+    -v $(pwd)/config.yaml:/config.yaml \
+    -v $(pwd)/data:/data \
+    -v /var/run/docker.sock:/var/run/docker.sock \
+    -e CONFIG_FILE=/config.yaml \
+    -e GITEA_INSTANCE_URL=<instance_url> \
+    -e GITEA_RUNNER_REGISTRATION_TOKEN=<registration_token> \
+    -e GITEA_RUNNER_NAME=<runner_name> \
+    -e GITEA_RUNNER_LABELS=<runner_labels> \
+    --name my_runner \
+    -d gitea/act_runner:nightly
+```
+
+You may notice that we have mounted the `/var/run/docker.sock` into the container.
+It is because the act runner will run jobs in docker containers, so it needs to communicate with the docker daemon.
+As mentioned, you can remove it if you want to run jobs in the host directly.
+To be clear, the "host" actually means the container which is running the act runner now, instead of the host machine.
+
+### Labels
+
+The labels of a runner are used to determine which jobs the runner can run, and how to run them.
+
+The default labels are `ubuntu-latest:docker://node:16-bullseye,ubuntu-22.04:docker://node:16-bullseye,ubuntu-20.04:docker://node:16-bullseye,ubuntu-18.04:docker://node:16-buster`.
+It is a comma-separated list, and each item is a label.
+
+Let's take `ubuntu-22.04:docker://node:16-bullseye` as an example.
+It means that the runner can run jobs with `runs-on: ubuntu-22.04`, and the job will be run in a docker container with the image `node:16-bullseye`.
+
+If you think the image is too poor, and you have enough disk space to use a better and bigger one, you can change it to `ubuntu-22.04:docker://<the image you like>`.
```

Comment:
What is meant by "poor"?

Reply:
Likely "if the default image is insufficient for your needs" is more appropriate

Reply:
What I tried to say is that the default image is small but it lacks a lot of tools, so it's "poor".

I think techknowlogick's suggestion is great.

```suggestion
If the default image is insufficient for your needs, and you have enough disk space to use a better and bigger one, you can change it to `ubuntu-22.04:docker://<the image you like>`.
```